angular-code-input/src/lib/code-input.component.ts

```
@@ -65,16 +66,19 @@ export class CodeInputComponent implements AfterViewInit, OnInit, OnChanges, OnD
     Object.assign(this, defaultComponentConfig);
 
-    if (!config) {
+    if (!this.config) {
       return;
     }
 
     // filtering for only valid config props
-    for (const prop in config) {
-      if (!config.hasOwnProperty(prop)) {
+    for (const prop in this.config) {
+      if (!this.config.hasOwnProperty(prop)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)